xapian-applications/omega/templates/activatelog

```diff
@@ -0,0 +1,2 @@
+$set{clicklog,true}
+$include{query}
```

Comment:
Sorry - I missed this before, but there shouldn't be a newline (or anything which gets output to the browser) before `$include{query}`.  If there is that means that the `$httpheader{Content-Type,text/html; charset=utf-8}` at the start of the `query` template can't work, because `$httpheader` can only work before anything has been output.